src/gsi/crtm_interface.f90

```diff
@@ -1762,6 +1795,38 @@ subroutine call_crtm(obstype,obstime,data_s,nchanl,nreal,ich, &
 
          cf(k)=min(max(zero,cf(k)),one)
      endif ! cf
+
+! Space-time interpolation of cloud fraction (cf)
+     if (n_clouds_fwd_wk>0 .and. inis==0) then
+         ni(k)=((niges_itsig (ix ,iy ,k)*w00+ &
+                 niges_itsig (ixp,iy ,k)*w10+ &
+                 niges_itsig (ix ,iyp,k)*w01+ &
+                 niges_itsig (ixp,iyp,k)*w11)*dtsig + &
+                (niges_itsigp(ix ,iy ,k)*w00+ &
+                 niges_itsigp(ixp,iy ,k)*w10+ &
+                 niges_itsigp(ix ,iyp,k)*w01+ &
+                 niges_itsigp(ixp,iyp,k)*w11)*dtsigp)
+
+!        Ensure ozone is greater than ozsmall
+
+         ni(k)=max(zero,ni(k))
+     endif ! ni
+
+! Space-time interpolation of cloud fraction (cf)
+     if (n_clouds_fwd_wk>0 .and. inrs==0) then
+         nr(k)=((nrges_itsig (ix ,iy ,k)*w00+ &
+                 nrges_itsig (ixp,iy ,k)*w10+ &
+                 nrges_itsig (ix ,iyp,k)*w01+ &
+                 nrges_itsig (ixp,iyp,k)*w11)*dtsig + &
+                (nrges_itsigp(ix ,iy ,k)*w00+ &
+                 nrges_itsigp(ixp,iy ,k)*w10+ &
+                 nrges_itsigp(ix ,iyp,k)*w01+ &
+                 nrges_itsigp(ixp,iyp,k)*w11)*dtsigp)
+
+!        Ensure ozone is greater than ozsmall
```

Comment:
Same question as above.  Is this comment correct?  The code below enforces a lower bound of `zero` on `nr`.